stacks/container-apps/feeder.prx.org.yml

```diff
@@ -78,6 +79,10 @@ Parameters:
   HealthCheckPath:
     Type: String
     Default: /api/v1
+  SharedRdsPostgresqlEndpoint:
```

Comment:
Is this needed?

Reply:
I wired it through, thinking we'd switch Prod over to using that param now.  And use the `Fn::If` to toggle prod to RDS and stag to Aurora.

But maybe not?

Reply:
No that makes sense.